### PR TITLE
Add script to add/update required filters. 

### PR DESCRIPTION
**Pull request by juniset**
- Remove filter for cSAI and cREP
- Add filters for cUNI and cUSDT
- Add filter for WETH
- Request filter update for Paraswap
- Request filter update for ParaswapUniV2RouterFilter (Uniswap v2)
- Request filter update for UniswapV2UniZapFilter

### CODE REVIEW

File: deployment/10_patch_filters.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  // //////////////////////////////////
+
+  console.log("Setting the deployment account as the owner of default registry");
+  await multisigExecutor.executeCall(DappRegistryWrapper, "changeOwner", [0, deploymentAccount]);
+
+  // //////////////////////////////////
+  // Update existing filters to Argent Registry
+  // //////////////////////////////////
+
+  const filters = {};
```
Comment (olivdb) [inline deployment/10_patch_filters.js:45]: I think you need to initialize it with existing filters:
```
const filters = {...config.filters};
```

File: deployment/10_patch_filters.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+  //
+  // Compound
+  //
+  for (const [underlying, cToken] of Object.entries(config.defi.compound.markets)) {
+    if (underlying === "0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359" || underlying === "0x1985365e9f78359a9B6AD760e32412f4a445E862") {
+      // remove SAI and REP filter
+      console.log(`Removing filter for Compound Underlying ${underlying}`);
+      await DappRegistryWrapper.removeDapp(0, cToken);
```
Comment (olivdb) [inline deployment/10_patch_filters.js:54]: Shouldn't we remove the corresponding filters from the `filters.CompoundFilter` array?

**Review by olivdb — APPROVED** [review]

**Comment by Muzcool** [comment]
👌

**Comment by Muzcool** [comment]
👌

File: deployment/10_patch_filters.js (hunk 2 above)
Comment (Muzcool) [reply deployment/10_patch_filters.js:54]: 🤝

**Pull request merged**